magefile.go

```diff
@@ -210,3 +215,23 @@ func Clean() error {
 
 	return os.Remove("package-registry")
 }
+
+func Vendor() error {
+	fmt.Println(">> mod - updating vendor directory")
+
+	err := sh.RunV("go", "mod", "tidy")
```

Comment:
Perhaps we can remove tidy in this PR so it doesn't remove the go-junit part?

Reply:
I think I solved this, see above.